# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/ScriptTrace.h
# Commit: 0527d227c6b6
# Full Hash: 0527d227c6b6f3d6648704a01bfd1c090b971ea7
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 21:52:29
# Regressor Bug: 1742437
# File Overlap Count: 2
# Description:
#   Bug 1742437 - Move module loader files into a central folder; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D137532
# ==============================================================================

diff -r 967e68ed5f56 -r 0527d227c6b6 dom/script/ScriptTrace.h
--- a/dom/script/ScriptTrace.h	Fri Feb 18 12:45:52 2022 +0000
+++ b/dom/script/ScriptTrace.h	Fri Feb 18 12:45:52 2022 +0000
@@ -7,8 +7,6 @@
 #ifndef mozilla_dom_ScriptTrace_h
 #define mozilla_dom_ScriptTrace_h
 
-#include "ScriptLoader.h"
-
 #include "mozilla/AsyncEventDispatcher.h"
 #include "mozilla/StaticPrefs_dom.h"
 